webapp/app.py

```diff
@@ -22,3 +25,32 @@
 template_finder_view = TemplateFinder.as_view("template_finder")
 app.add_url_rule("/", view_func=template_finder_view)
 app.add_url_rule("/<path:subpath>", view_func=template_finder_view)
+
+
+def static_url(filename):
```

Comment:
This funciton can be removed since the flask base has: https://github.com/canonical-web-and-design/canonicalwebteam.flask-base/blob/8e303653e442d87937a1c89398d0594f17ecbf73/canonicalwebteam/flask_base/context.py#L15